dist/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+<script defer src="main.js"></script></head>
+<body>
+    <main class="d-flex flex-col align-center">
+        <h1>Leaderboard</h1>
+        <section class="d-flex justify-center w-100">
+            <div class="p-20">
+                <div>
+                    <h2 class="d-inline me-10">Recent scores</h2>
+                    <button class="btn d-inline pos-rel btn-shadow">Refresh</button>
+                </div>
+                <table class="d-block mt-10 border">
+                    <tbody>
+                        <tr><td>Name: 100</td></tr>
+                        <tr><td>Name: 20</td></tr>
+                        <tr><td>Name: 50</td></tr>
+                        <tr><td>Name: 78</td></tr>
+                        <tr><td>Name: 125</td></tr>
+                        <tr><td>Name: 77</td></tr>
+                        <tr><td>Name: 42</td></tr>
+                    </tbody>
+                </table>
+            </div>
+            <div class="p-20">
+                <h2 class="d-inline">Add your score</h2>
+                <form>
+                    <input class="mt-10 border" placeholder="Your Name"/>
+                    <br />
+                    <input class="mt-10 border" placeholder="Your Score"/>
+                    <br />
+                    <button class="btn mt-10 float-right pos-rel btn-shadow">Submit</button>
+                </form>
+            </div>
+        </section>
+    </main>
```

Comment:
- [ ] In place to use HTML code to create this part, try to use index.js to create all this dynamically with js code. I recommend you create `<div id="root"></div>` and call it on `index.js` to create all the elements inside it.
- Don't forget to use ES6 best practices on it.

Reply:
Hi, thank you for the review, however, we were asked to use html to structure our code in this milestone which is why I manually created it this way.

Reply:
Hi there, thanks for telling me this, I'll share the entire text at that point, it said `Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:`. I invite you to add the JS code to the next review.
- Hint: index.html main contain only root element.